deploy/olm-catalog/service-telemetry-operator/manifests/infra.watch_servicetelemetrys_crd.yaml

```diff
@@ -149,6 +149,72 @@ spec:
                       type: object
                   type: object
               type: object
+            logs:
+              description: Logs related backend configuration.
+              properties:
+                loki:
+                  description: Logs storage backend Loki
+                  properties:
+                    enabled:
+                      description: Enable Loki as a storage backend for logs
+                      type: boolean
+                    replicationFactor:
+                      description: Loki replication factor
+                      type: string
+                    flavor:
+                      description: Loki flavor
+                      enum:
+                      - 1x.extra-small
+                      - 1x.small
+                      - 1x.medium
+                      type: string
+                    storage:
+                      description: Logs storage configuration for Loki
+                      properties:
+                        objectStorageSecret:
+                          description: Secret containing informaiton required for S3 object storage
+                          type: string
+                        storageClassName:
```

Comment:
I noted earlier that you're calling `storageClass` in the manifest, but that you've defined this as `storageClassName` here. I think actually this should be renamed to `storageClass` liek the other `storage...` properties in this file for consistency.